tests/test_load_aircraft_csv.py

```diff
@@ -0,0 +1,180 @@
+import datetime
+import os
+import unittest
+
+from importers.aircraft_csv_format_importer import AircraftCsvFormatImporter
+from pepys_import.core.store.data_store import DataStore
+from pepys_import.file.file_processor import FileProcessor
+from tests.utils import check_errors_for_file_contents
+
+FILE_PATH = os.path.dirname(__file__)
+DATA_PATH_SHORT = os.path.join(FILE_PATH, "sample_data/track_files/csv_data/aircraft/OwnPos_UC.csv")
+DATA_PATH_LONG = os.path.join(
+    FILE_PATH, "sample_data/track_files/csv_data/aircraft/OwnPos_UC_Long.csv"
+)
+DATA_PATH_MISSING = os.path.join(
```

Comment:
DATA_PATH_MISSING isn't used. It appears you aren't testing the file for missing data.

Reply:
This was accidentally left over from my initial plan but was replaced by calls to `check_errors_for_file_contents` which now checks for an incorrect or missing format in each of the cell types to ensure that they're handled correctly. As a result, I don't think this is needed anymore and so I can probably remove it, unless you think it's still worth testing on a document as well as individual lines?

Reply:
I think that just using check_errors_for_file_contents is fine